### ISSUE TITLE
Missing JS library

### ISSUE THREAD

**Issue by mrbech** [vpavkin/scala-js-momentjs#24]
After a clean build I suddenly get the following errors at compile time:
```
org.scalajs.core.tools.jsdep.JSLibResolveException: Some references to JS libraries could not be resolved:
- Missing JS library: moment/2.15.1/min/moment-with-locales.js
  originating from: scalaJsMomentJs:compile
- Missing JS library: 2.15.1/min/moment-with-locales.js
  originating from: scalaJsMomentJs:compile
```

**Comment by vpavkin** [comment]
Hi @mrbech. This happened to me as well. Do you have explicit jsDependency on moment.js in your build?
I would suggest trying to set versions explicitly in `dependsOn` clause, like this:

``` scala
val momentVersion = "2.15.1"
 jsDependencies += 
      "org.webjars.npm" % "moment" % momentVersion
        / s"$momentVersion/moment.js"
        minified s"$momentVersion/min/moment.min.js"
```

**Comment by vpavkin** [comment]
Also I noticed that 2.15.2 is out, will release a new version soon

**Comment by mrbech** [comment]
I have no explicit jsDependency on moment.js in my build, was not an issue earlier. Do I need one now?

**Comment by vpavkin** [comment]
issue is that for some reason, latest moment.js is always downloaded along with the version specified in the facade. When facade is not on the latest version, it means that there are two versions downloaded and ScalaJS can't really choose.

I see two options here:

1) you just migrate on fresher facade version (which I'll release in 30 min). But this will break again when new moment comes out.
2) you specify moment version explicitly and add an explicit dependency (I do it this way in my work project). This will not break.

I will investigate if anything else can be done here, but this won't be quick.

**Comment by vpavkin** [comment]
Just released 0.3.3 based on 2.15.2 to sonatype, should be on maven soon. Should fix your problem.

I think I see the cause in facade's build.sbt, will check it after work :)

**Comment by mrbech** [comment]
Awesome, works :) Thanks for the quick response!
